src/example-types/metricGroups/pivot.js

```diff
@@ -109,22 +109,21 @@ let buildQuery = async (node, schema, getStats) => {
     let sortField = getSortField(sort)
 
     return _.reduce(
-      async (children, group) => {
-        // Calculating subtotal metrics at each group level if not flattening or drilling down
-        // Support for per group stats could also be added here - merge on another stats agg blob to children based on group.stats/statsField or group.values
-        if (!node.flatten && !node.drilldown)
-          children = _.merge(await children, statsAggs)
+      async (children, row) => {
```

Comment:
everything in buildNestedGroupQuery is groups and not rows

Reply:
Fixed